### PR TITLE
Add OneNote Plugin

### PR DESCRIPTION
**Pull request by Odotocodot**

### CODE REVIEW

**Comment by taooceros** [comment]
It's a cool plugin! Thanks

**Review by taooceros — APPROVED** [review]

**Comment by Odotocodot** [comment]
Hopefully it should pass the workflows now!

File: plugins.json [hunk 1]
```diff
@@ -1239,5 +1239,18 @@
         "IcoPath": "https://cdn.jsdelivr.net/gh/garulf/MC-Multi-Launcher@main/src/icon.png",
         "DateAdded": "2022-12-15T07:23:44Z",
         "LatestReleaseDate": "2022-12-15T05:33:19Z"
+    },
+    {
+        "ID": "506B2EE8-4117-4BDF-8F00-DF58498DF922",
+        "ActionKeyword": "on",
```
Comment (jjw24) [inline plugins.json:1245]:
```suggestion
```

**Review by Odotocodot — COMMENTED** [review]
Ahhh I see, thanks for the fix! Totally missed the view changes button beforehand whoops, still getting used to github.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**